properties/size/border-radius.json

```diff
@@ -0,0 +1,12 @@
+{
+  "size": {
+    "border-radius": {
+      "circle": { "value": "50%" },
```

Comment:
@juanfabrega so here's a new scenario. i want this value to be a percentage rather than rems. How do we handle this?

Here's what `yarn build` outputs now
<img width="369" alt="Screen Shot 2020-07-15 at 4 29 05 PM" src="https://user-images.githubusercontent.com/1447339/87609769-4e68e200-c6b8-11ea-962a-4e5c061c97ba.png">